modules/virt-define-http-liveness-probe.adoc

```diff
@@ -4,74 +4,36 @@
+
+Define an HTTP liveness probe by setting the `spec.livenessProbe.httpGet` field of the virtual machine instance (VMI) configuration.
 
-This procedure provides an example configuration file for defining HTTP
-liveness probes.
 
 .Procedure
 
-. Customize a YAML configuration file to create an HTTP liveness probe, using
-the following code block as an example. In this example:
-+
---
-* You configure a probe using `spec.livenessProbe.httpGet`, which queries port `1500` of the virtual machine instance, after an initial delay of `120` seconds.
-* The virtual machine instance installs and runs a minimal HTTP server
-on port `1500` using `cloud-init`.
---
-+
-[NOTE]
-====
-The `timeoutSeconds` value must be lower than the `periodSeconds` value. The `timeoutSeconds` default value is `1`. The `periodSeconds` default value is `10`.
-====
+. Edit the VMI configuration file to include details of the HTTP liveness probe.
 +
+
+.Sample liveness probe with an HTTP GET test 
 [source,yaml]
 ----
-apiVersion: kubevirt.io/v1alpha3
-kind: VirtualMachineInstance
-metadata:
-  labels:
-    special: vmi-fedora
-  name: vmi-fedora
 spec:
-  domain:
-    devices:
-      disks:
-      - disk:
-          bus: virtio
-        name: containerdisk
-      - disk:
-          bus: virtio
-        name: cloudinitdisk
-    resources:
-      requests:
-        memory: 1024M
   livenessProbe:
-    initialDelaySeconds: 120
-    periodSeconds: 20
-    httpGet:
-      port: 1500
-    timeoutSeconds: 10
-  terminationGracePeriodSeconds: 0
-  volumes:
-  - name: containerdisk
-    registryDisk:
-      image: kubevirt/fedora-cloud-registry-disk-demo
-  - cloudInitNoCloud:
-      userData: |-
-        #cloud-config
-        password: fedora
-        chpasswd: { expire: False }
-        bootcmd:
-          - setenforce 0
-          - dnf install -y nmap-ncat
-          - systemd-run --unit=httpserver nc -klp 1500 -e '/usr/bin/echo -e HTTP/1.1 200 OK\\n\\nHello World!'
-    name: cloudinitdisk
+    initialDelaySeconds: 120 <1>
+    periodSeconds: 20 <2>
+    httpGet: <3>
+      port: 1500 <4>
+    timeoutSeconds: 10 <5>
 ----
-+
-. Create the virtual machine instance by running the following command:
+<1> The time, in seconds, after the VMI starts before the liveness probe is initiated.
+<2> The delay, in seconds, between performing probes. The default delay is 10 seconds. This value must be greater than `timeoutSeconds`.
+<3> The HTTP GET request to perform to connect to the VMI.
+<4> The port of the VMI that the probe queries. In the above example, the probe queries port 1500. The VMI installs and runs a minimal HTTP server on port 1500 via cloud-init.
+<5> The number of seconds of inactivity after which the probe times out and the VMI is assumed to have failed. The default value is 1. This value must be lower than `periodSeconds`.
+
+. Create the VMI by running the following command:
 +
 [source,terminal]
 ----
-$ oc create -f <file name>.yaml
+$ oc create -f <file_name>.yaml
```

Comment:
@kwiesmueller Regarding Andrew's first comment, I remember we talked about using the VM config instead of the VMI? Would that make more sense? 

@aburdenthehand As far as I understand, you cannot edit an existing VMI. The probe information would be added when first creating the VMI.